src/main/java/net/troja/eve/esi/api/SsoApi.java

```diff
@@ -70,8 +71,59 @@ public CharacterInfo getCharacterInfo() throws ApiException {
                     localVarHeaderParams, localVarFormParams, localVarAccept, localVarContentType, localVarAuthNames,
                     localVarReturnType);
         } finally {
-            apiClient.setBasePath(basePath); // load old basepath
             apiClient.setDateFormat(dateFormat); // load old date format
         }
     }
+
+    public void revokeRefreshToken(String refreshToken) throws ApiException {
+        revokeToken(refreshToken, REFRESH_TOKEN);
+    }
+
+    public void revokeAccessToken(String accessToken) throws ApiException {
+        revokeToken(accessToken, ACCESS_TOKEN);
+    }
+
+    private void revokeToken(String token, String tokenTypeHint) throws ApiException {
+        final String basePath = apiClient.getBasePath(); // Save old basepath
+        apiClient.setBasePath(URI_REVOKE); // Set new basepath
+
+        final Object localVarPostBody = null;
+
+        final String localVarPath = "/oauth/revoke";
+
+        final List<Pair> localVarQueryParams = new ArrayList<>();
+        
+        final Map<String, String> localVarHeaderParams = new HashMap<>();
+        final OAuth auth = (OAuth) apiClient.getAuthentication("evesso");
+        String clientPair =  auth.getClientId() + ":" + auth.getClientSecret();
+        String encoded;
+        try {
+            encoded = new String(Base64.getUrlEncoder().encode(clientPair.getBytes("UTF-8")));
+        } catch (UnsupportedEncodingException ex) {
+            throw new ApiException(ex);
+        }
+        localVarHeaderParams.put("Authorization", "Basic " + encoded);
+        localVarHeaderParams.put("Content-Type", "application/x-www-form-urlencoded");
+        localVarHeaderParams.put("Host", "login.eveonline.com");
+
+        final Map<String, Object> localVarFormParams = new HashMap<>();
+        localVarFormParams.put("token_type_hint", tokenTypeHint);
+        localVarFormParams.put("token", token);
+
+        final String[] localVarAccepts = { "application/json" };
+        final String localVarAccept = apiClient.selectHeaderAccept(localVarAccepts);
+
+        final String[] localVarContentTypes = { "application/x-www-form-urlencoded" };
+        final String localVarContentType = apiClient.selectHeaderContentType(localVarContentTypes);
+
+        final String[] localVarAuthNames = new String[0];
+
+        try {
+            apiClient.invokeAPI(localVarPath, "POST", localVarQueryParams, localVarPostBody,
```

Comment:
replace localVarPostBody with null (constant)?  compiler will do this anyway, but looks like this was left over from previous version.